src/validations/noFakeLists.js

```diff
@@ -0,0 +1,29 @@
+
+/**
+ * @param {Block[]} blocks - Portable Text AST
+ * @param {Object} [options = {}] - Options for the validation
+ * @param {RegExp} [options.regex = /\n\s*[-*+–—]/] - Validation regex
+ * @param {String} [options.message = "This looks like a list, but it is plain text. Use the bulleted list option."] - Message to display when the validation triggers
+ */
+
+function noFakeLists(blocks, options = {}) {
+  const regex = options.regex || /\n\s*[-*+–—]/;
+  const fakeListBlocks = (blocks || []).filter(
+    block =>
+      block._type === 'block' &&
+      block.children.some(blockPart =>
+        blockPart._type === 'span' &&
+        blockPart.text.match(regex)
```

Comment:
Considering we’re only interested in whether or not the string matches the regular expression, `RegExp.prototype.test` is probably better than `String.prototype.match`, particularly in terms of performance ([ref](https://stackoverflow.com/a/10940138)).

```suggestion
        regex.test(blockPart.text)
```